todo-app/src/TodoList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Todo from './Todo';
+
+
+const todos = [
+ {
+   id: '1',
+   task: "Get out of bed",
+   time: "Wed Sep 13 2017"
+ },
+ {
+  id: '2',
+  task: "Brush teeth",
+  time: "Thu Sep 14 2017"
+ },
+ {
+  id: '3',
+  task: "Eat breakfast",
+  time: "Fri Sep 15 2017" 
+ }
+]
+
+export default class TodoList extends React.Component {
+    render() {
```

Comment:
It is customary to have the `render` function be the _last_ function in a React component. Of course this is 100% a matter of preference, but it's how most devs do it. You can check out [this link](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md) for examples within an `eslint` rule.